server/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -547,6 +548,8 @@ private Modal blockDescriptionModal(Tag csrfTag, BlockForm blockForm, String blo
         .withId("block-edit-form")
         .with(
             div(
+                    h3("The following fields will only be visible to administrators")
```

Comment:
Do we distinctly know what H level is above this?

a11y needs correct hierarchy.

Not that we have that as is, but being mindful here if we can do the right thing.

Reply:
Good point. I made it an h1 in this case since it's a modal dialog. Since it is contained within a modal, the main visible content to the screen reader would be this heading. As such, it should be h1. Used TW classes to decrease the size.